src/schema/objects/columns.yaml

```diff
@@ -45,6 +45,14 @@ cardiac:
   description: |
     continuous pulse measurement
   type: number
+channel:
+  name: channel
+  display_name: Channel
+  description: |
+    Channel(s) associated with an event.
+    If multiple channels are specified, separate with commas (`,`) or a
+    delimiter specified in the `"Delimiter"` field describing the `channel`
+    column.
```

Comment:
seems like a type needs to be declared for this entry?

> ValueError: Type could not be inferred for channel

https://app.circleci.com/pipelines/github/bids-standard/bids-specification/6246/workflows/b6e08071-be84-4391-b1a8-e3ae6bd28ac0/jobs/18265